### DISCUSSION TITLE
Error in building Delays project

### DISCUSSION THREAD

**Discussion by Bader-EE** [Mair/esp32-course#127] [Course Qeustions] [answered]
[Problem solved accedintaly]

Hello Mr. Mair,

I hope this question finds you well.

> The ESP32 course
> Section [3]: A First Look - Introduction 
> Video [3]: Delays

I tried to write the same code but have **errors in building** prosses [the errors are shown in the attached picture].

<img width="960" alt="error2" src="https://user-images.githubusercontent.com/108470079/177739287-a37b84a0-fb9d-41de-ba85-42e7214ae5da.png">

**Comment by Mair** [comment, marked as answer]
Hi @Bader-EE 
The folder you are working with is not allowed which is causing the compilation error.
the directory path should not have spaces and should not start with a number
change the directory "1- Delay_project" to _1-Delay_project, then do an idf.py fullclean and then try to rebuild and you should be good to go
